docs/spec/spec.md

```diff
@@ -129,41 +130,43 @@ The conforming implementation of the specification is released and included in t
     * 11.3 [Execution](#113-execution)
         * 11.3.1 [Service Interceptors](#1131-service-interceptors)
         * 11.3.2 [Field Interceptors](#1132-field-interceptors)
-12. [Security](#12-security)
-    * 12.1 [Service Authentication and Authorization](#121-service-authentication-and-authorization)
-        * 12.1.1 [Declarative Approach](#1211-declarative-approach)
-            * 12.1.1.1 [Basic Authentication - File User Store](#12111-basic-authentication---file-user-store)
-            * 12.1.1.2 [Basic Authentication - LDAP User Store](#12112-basic-authentication---ldap-user-store)
-            * 12.1.1.3 [JWT Authentication](#12113-jwt-authentication)
-            * 12.1.1.4 [OAuth2](#12114-oauth2)
-        * 12.1.2 [Imperative Approach](#1212-imperative-approach)
-            * 12.1.2.1 [Basic Authentication - File User Store](#12121-basic-authentication---file-user-store)
-            * 12.1.2.2 [Basic Authentication - LDAP User Store](#12122-basic-authentication---ldap-user-store)
-            * 12.1.2.3 [JWT Authentication](#12123-jwt-authentication)
-            * 12.1.2.4 [OAuth2](#12124-oauth2)
-    * 12.2 [Client Authentication and Authorization](#122-client-authentication-and-authorization)
-        * 12.2.1. [Basic Authentication](#1221-basic-authentication)
-        * 12.2.2. [Bearer Token Authentication](#1222-bearer-token-authentication)
-        * 12.2.3. [Self-Signed JWT Authentication](#1223-self-signed-jwt-authentication)
-        * 12.2.4. [OAuth2](#1224-oauth2)
-            * 12.2.4.1 [Client Credentials Grant Type](#12241-client-credentials-grant-type)
-            * 12.2.4.2 [Password Grant Type](#12242-password-grant-type)
-            * 12.2.4.3 [Refresh Token Grant Type](#12243-refresh-token-grant-type)
-            * 12.2.4.4 [JWT Bearer Grant Type](#12244-jwt-bearer-grant-type)
-    * 12.3 [SSL/TLS and Mutual SSL](#123-ssltls-and-mutual-ssl)
-        * 12.3.1 [Listener](#1231-listener)
-            * 12.3.1.1 [SSL/TLS](#12311-ssltls)
-            * 12.3.1.2 [Mutual SSL](#12312-mutual-ssl)
-        * 12.3.2 [Client](#1232-client)
-            * 12.3.2.1 [SSL/TLS](#12321-ssltls)
-            * 12.3.2.2 [Mutual SSL](#12322-mutual-ssl)
-13. [Federation](#13-federation)
-    * 13.1 [Federated Subgraph](#131-federated-subgraph)
-        * 13.1.1 [The `@subgraph:Subgraph` Annotation](#1311-the-graphqlsubgraph-annotation)
-        * 13.1.2 [The `@subgraph:Entity` Annotation](#1312-the-graphqlentity-annotation)
-        * 13.1.3 [The `subgraph:ReferenceResolver`](#1311-the-referenceresolver)
-14. [Tools](#14-tools)
-    * 14.1 [GraphiQL Client](#141-graphiql-client)
+12. [Constraint Validation](#12-constraint-validation)
```

Comment:
This means we have to update the links in the BBEs, etc. Can you check them?

Reply:
Oh! missed that, It seems we have to change the spec link in multiple BBEs. In that case, shall add this after the `federation` section?

Reply:
As discussed offline, A new section named `Additional Features`(or maybe we can have a better name) can be added to the spec. Under this new section, we can add `File Upload, Interceptors, Constraint Validation`. If we are adding this, it's better to handle this in a separate issue. @ThisaruGuruge  WDYT?

Reply:
+1

Reply:
Let's wait for the Update 6 release and revisit this. Can you create an issue to track the progress?

Reply:
created an issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/4466

Reply:
Let's revert these changes for now.